### DISCUSSION TITLE
Will the Appium flutter driver switch to use integration_test instead of flutter_driver

### DISCUSSION THREAD

**Discussion by jfortier882** [appium-userland/appium-flutter-driver#661] [General]
I understand that the flutter_driver is or soon will be deprecated. Will this driver be updated soon to use the integration_test package?

**Comment by KazuCocoa** [comment]
No concrete plan. Mostly, this is because no much time to investigate the possibility. We're welcome to contribute if anyone has already done some kind of implementation/investigation.

As https://github.com/appium/appium-flutter-driver?tab=readme-ov-file#appium-flutter-driver-or-appium-uiautomator2xcuitest-driver and flutter 3.19, Flutter provides more possibilities for uia2/xcuitest only way as well.

**Reply by KazuCocoa** [reply]
It looks like this kind of discussion is still ongoing, so `flutter_driver` could continue to be maintained. Let's keep watching relevant issue in Flutter repo such as https://github.com/flutter/flutter/issues/148028

**Comment by saikrishna321** [comment]
We have new driver based in integration test - https://github.com/AppiumTestDistribution/appium-flutter-integration-driver
